main.js

```diff
@@ -41,7 +149,6 @@ document.getElementById("startButton").addEventListener("click", function(){
   ];
 document.getElementById("title").addEventListener("click", function(){
```

Comment:
I don't see any element in the HTML with the `title` id. Is this used?

Reply:
I have a **h1** tag with the title id. It's <h1 id="title">Bounce</h1>